app/renderer/network.html

```diff
@@ -1,16 +1,25 @@
 <!DOCTYPE html>
 <html lang="en" class="responsive desktop">
-    <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-        <meta name="viewport" content="width=device-width">
-        <title>Zulip - Network Troubleshooting</title>
-        <link rel="stylesheet" href="css/network.css" type="text/css" media="screen">
-    </head>
-    <body>
+
+<head>
```

Comment:
Please indent the `<head>` block. If you look at the diff, you should notice a difference.

Reply:
Tried doing this, now linting tests are failing. I tried going back to the old code version but linting fails there also. I guess tabs are not being considered as 4 spaces if i'm not wrong